lib/hawkular/inventory/inventory_api.rb

```diff
@@ -429,6 +429,17 @@ def create_metric_type(feed_id, metric_type_id, type = 'GAUGE', unit = 'NONE', c
       MetricType.new(new_mt)
     end
 
+    # List operation types for given resource type
+    # @param [String] resource_type_path canonical path of the resource type entity
+    # @return [Array<String>] List of operation type ids
```

Comment:
I don't get your comment :] Array is normal data structure in Ruby http://docs.ruby-lang.org/en/2.0.0/Array.html  what's wrong with returning an array of stuff in Ruby?